src/lib.rs

```diff
@@ -816,6 +817,9 @@ impl<'a> ParserState<'a> {
             if self.consume(b"$C") {
                 sc = self.read_qualifier();
             }
+            if self.consume(b"$V") {
+                return Ok(Type::EmptyParameterPack);
```

Comment:
LLVM will use $$V here as well depending on the ABI version. We should probably support both.